libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -1,18 +1,43 @@
-import { Component } from '@angular/core';
+import { Component, OnDestroy } from '@angular/core';
 import { Store } from '@ngrx/store';
-import { getReadingList, removeFromReadingList } from '@tmo/books/data-access';
+import { addToReadingList, getReadingList, removeFromReadingList} from '@tmo/books/data-access';
+import { MatSnackBar } from "@angular/material/snack-bar";
+import { Book, ReadingListItem } from "@tmo/shared/models";
+import { Subscription } from "rxjs";
 
 @Component({
   selector: 'tmo-reading-list',
   templateUrl: './reading-list.component.html',
   styleUrls: ['./reading-list.component.scss']
 })
-export class ReadingListComponent {
+export class ReadingListComponent implements OnDestroy{
   readingList$ = this.store.select(getReadingList);
+  private snackBarAction$: Subscription;
 
-  constructor(private readonly store: Store) {}
+  constructor(
+      private readonly store: Store,
+      private readonly snackBar: MatSnackBar
+  ) {}
 
   removeFromReadingList(item) {
     this.store.dispatch(removeFromReadingList({ item }));
+    this.displaySnackBar(item);
```

Comment:
Here you are just showing the snackbar at the same time that the `removeFromReadingList` action is dispatched. What if the action fails? I think you should show the snackbar after the `removeFromReadingList` action is confirmed!